utils.py

```diff
@@ -16,6 +16,14 @@ def trn_to_numpy(filename):
         return numpy_array.reshape(img.shape[0], img.shape[1])
 
 
+def numpy_to_trn(filename, mask):
```

Comment:
@Foxiy I believe this should just return the contents of the `.trn` file (i.e. the line 23). Saving the file should probably happen somewhere else.